raiwidgets/tests/model_analysis.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+from raiwidgets import ModelAnalysisDashboard
+from responsibleai import ModelAnalysis
+import sklearn
+import shap
```

Comment:
Is shap a dependency?

Reply:
through interpret-community
I don't want to introduce extra dependency because of one dataset.
If this is concern we can check if other dataset is good to use.
For now this file is just for manual testing, we don't have automated on it.